src/WebJobs.Extensions.DurableTask/ContextInterfaces/IDurableOrchestrationClient.cs

```diff
@@ -279,6 +289,13 @@ public interface IDurableOrchestrationClient
         /// <returns>Returns an instance of <see cref="PurgeHistoryResult"/>.</returns>
         Task<PurgeHistoryResult> PurgeInstanceHistoryAsync(string instanceId);
 
+        /// <summary>
+        /// Purge the history for multiple instances.
+        /// </summary>
+        /// <param name="instanceIds">The IDs of the orchestration instance to purge.</param>
```

Comment:
Minor typo: change `instance` to `instances`
```suggestion
        /// <param name="instanceIds">The IDs of the orchestration instances to purge.</param>
```